docs/_guide/start.md

```diff
@@ -8,131 +8,312 @@ slug: start
 
-**Refer to third-party component documentation first.** To work with any existing component made by a third party, see its documentation. This guide should work for most LitElement-based components if they are published on npm.
+### Install LitElement
 
-Many components are published on npm and can be installed from the command line:
+LitElement is available from npm:
 
 ```bash
-cd my-project-folder
-npm install package-name
+npm i lit-element 
 ```
 
-In an HTML document, a component published on npm can be imported from the `node_modules` folder:
+### Add an element
 
-```html
-<head>
-  <script type="module" src="node_modules/package-name/existing-element.js"></script>
-</head>
-<body>
-  <existing-element></existing-element>
-</body>
-```
+Pick a location to store your LitElement components, and create a test element: 
 
-To import into another JavaScript module, use the component's package name:
+_components/my-element.js_
 
 ```js
-import 'package-name/existing-element.js';
+import {LitElement, html} from 'lit-element';
 
-class MyElement extends LitElement{
-  render(){
+class MyElement extends LitElement {
+  render() {
     return html`
-      <existing-element></existing-element>
+      <div>Hello from MyElement!</div>
     `;
   }
 }
+
 customElements.define('my-element', MyElement);
 ```
+
+Import your component:
+
+Index.js
+
+```js
+import './components/my-elements.js';
+```
+
+Use your component:
+
+index.html
+
+```html
+<my-element></my-element>
+```
+
+At this point, you should be able to build and run your project and see the "Hello from MyElement!" message.
+
+<div class="alert alert-info">
+
+**Adjust for your build system.** How you import the component may vary slightly depending on your build system and project structure. 
+
+</div>
+
+
+### Optional: Use ES dev server
+
+If you already have a dev server that works with your build system, it should work with LitElement. [ES dev server](https://open-wc.org/developing/es-dev-server.html) is an alternative dev server that provides a simple, bundler-free workflow. It performing a small number of useful transforms, including transforming bare module specifiers, transpiling to ES5 when needed, and adding polyfills for older browsers.
+
+1. Install es-dev-server
+
+    ```bash
+    npm i -D es-dev-server
+    ```
+
+1. Add a dev server command to `package.json`: 
+
+
+    ```json
+    {
+      "scripts": {
+        "start": "es-dev-server --app-index index.html --node-resolve --watch --open"
+      }
+    }
+    ```
+
+1. Start ES dev server:
+
+    ```bash
+    npm run start
+    ```
+
+### Supporting older browsers
+
+To support older browsers that don't support ES6 and the web components specifications, you'll need to take a few extra steps:
+
+*   Compile to ES5. Your build will also need to transform JavaScript modules into a format compatible with older browsers, like AMD. 
```

Comment:
Please remember that Babel users will need extra hint on using `babel.config.js` to transpile `node_modules`

See https://github.com/Polymer/lit-element/issues/883#issuecomment-577651183

Reply:
I think it's worth considering recommending [SystemJS](https://github.com/systemjs/systemjs) instead of AMD. AMD is a different module format with different rules. You can map most of es modules to it, but not all. SystemJS tries to implement the full es module spec, so you can be certain your code will work correctly when using things like live bindings, import.meta and soon import maps.

Reply:
Addressed by the same set of changes as above.